common/thorhelper/thorcommon.cpp

```diff
@@ -2097,6 +2098,7 @@ static bool getTranslators(Owned<const IDynamicTransform> &translator, Owned<con
                 if (keyedTranslator && (sourceFormat != expectedFormat))
                 {
                     Owned<const IKeyTranslator> _keyedTranslator = createKeyTranslator(sourceFormat->queryRecordAccessor(true), expectedFormat->queryRecordAccessor(true));
+                    //MORE: What happens if the key filters cannot be translated?
```

Comment:
Has this question been answered? Is there a Jira to revisit it?

Reply:
There is a jira HPCC-22287.  Currently we don't spot potential problems.